### PR TITLE
Grasp pose rotation

### PR DESCRIPTION
**Pull request by davetcoleman**
This was a lingering todo in the code:

```
// turn on Z axis //TODO parametrize this
```

That allows the Clam arm to have its grasps rotated 90 deg based on different robot's different geometry. 

@bmagyar i didn't know if you might want to review

### CODE REVIEW

File: src/simple_grasps.cpp [hunk 1]
```diff
@@ -190,7 +190,7 @@ bool SimpleGrasps::generateAxisGrasps(
 
 
     // DEBUG - show original grasp pose before tranform to gripper frame
-    if( false )
+    if( true )
```
Comment (bmagyar) [inline src/simple_grasps.cpp]: when this was enabled, we experienced big bumps in the responsiveness of the planner, can you make this a startup param?

**Comment by bmagyar** [comment]
I made one comments, but apart from that it looks good!

I was experimenting with planning around Z lately, but I didn't really get the results I wanted so those PRs will have to wait a bit more.

**Comment by davetcoleman** [comment]
Good point. There is now an optional `verbose` flag that is by default false.

**Comment by davetcoleman** [comment]
Thanks for the review!

**Pull request merged**